sentry-android-core/api/sentry-android-core.api

```diff
@@ -280,6 +280,7 @@ public final class io/sentry/android/core/SentryAndroid {
 	public static fun init (Landroid/content/Context;Lio/sentry/ILogger;)V
 	public static fun init (Landroid/content/Context;Lio/sentry/ILogger;Lio/sentry/Sentry$OptionsConfiguration;)V
 	public static fun init (Landroid/content/Context;Lio/sentry/Sentry$OptionsConfiguration;)V
+	public static fun replay ()Lio/sentry/IReplayApi;
```

Comment:
Can we move this under `Sentry`? We don't use `SentryAndroid` anywhere except for init, so I think it'd be a bit awkward to start throwing `SentryAndroid` around as public API.